daemon/src/daemon.rs

```diff
@@ -24,7 +24,7 @@ pub struct AssetDaemon {
 impl Default for AssetDaemon {
     fn default() -> Self {
         Self {
-            db_dir: PathBuf::from(".amethyst"),
+            db_dir: PathBuf::from(".db"),

```

Comment:
I think `.db` is a bit too generic? Maybe something about assets or amethyst would be good.